README.md

```diff
@@ -160,38 +111,23 @@ Here are some of the planned improvements:
 - [x] [Windowed aggregations over Tumbling & Hopping windows](https://quix.io/docs/quix-streams/v2-0-latest/windowing.html)
 - [x] [State recovery based on Kafka changelog topics](https://quix.io/docs/quix-streams/advanced/stateful-processing.html#fault-tolerance-recovery)
 - [x] [Group-by operation](https://quix.io/docs/quix-streams/groupby.html)
-- [X] ["Exactly Once" delivery guarantees for Kafka message processing (AKA transactions)](https://quix.io/docs/quix-streams/configuration.html#processing-guarantees)
+- [x] ["Exactly Once" delivery guarantees for Kafka message processing (AKA transactions)](https://quix.io/docs/quix-streams/configuration.html#processing-guarantees)
 - [ ] Joins
 - [ ] Windowed aggregations over Sliding windows
 - [ ] Support for Avro and Protobuf formats
 - [ ] Schema Registry support
```

Comment:
I'd remove the bullet points as it looks a little odd; clashes with the check boxes. Also, I'm not a fan of the `x` as it can be confusing (does that mean "yes" or "no"?). Perhaps ✓ is more appropriate?

Also, if we have estimates on the roadmap it might be good to add them in parentheses, e.g. Joins (Q4 2024)